### ISSUE TITLE
Can't install on Apple Silicon

### ISSUE THREAD

**Issue by joaogui1** [grpc/grpc#28449]
Hi! I'm trying to install `grpcio` on an Macbook pro (14-inch, Monterey, M1 Max), but I seem to consistently get the following error:
```
Could not find <Python.h>. This could mean the following:
      * You're on Ubuntu and haven't run `apt-get install python3-dev`.
      * You're on RHEL/Fedora and haven't run `yum install python3-devel` or
        `dnf install python3-devel` (make sure you also have redhat-rpm-config
        installed)
      * You're on Mac OS X and the usual Python framework was somehow corrupted
        (check your environment variables or try re-installing?)
      * You're on Windows and your Python installation was somehow corrupted
        (check your environment variables or try re-installing?)
```
I have installed Google's JAX, which is a moderately complex package, with no problem, so I don't think the python installation is corrupted.
I have also tried the solutions on stackoverflow that add `GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 GRPC_PYTHON_BUILD_SYSTEM_ZLIB=1` before the pip incantation to no avail
Any idea what the issue could be and how to fix it? Is this a known bug?

**Comment by EZ4ZZW** [comment]
I install grpcio by pip get error, but successfully installed using `conda install grpcio`.

**Comment by lclpedro** [comment]
if doesn't resolved above, try install python 3.9.8 or superior..

**Comment by donnadionne** [comment]
This seems to be an issue with an individual environment.
